data/virt_autotest/guest_autoyast_xml_files/opensuse_tumbleweed_kvm_hvm_guest_x86_64.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0"?>
```

Comment:
Is there known differences between kvm hvm guest and xen pv/fv guest autoyast files? I do not compare in details, but I do not find any. If yes, we should keep them all. Otherwise, it will be better to reuse one autoyast file to avoid duplication.

Reply:
there are two differences among these autoyast templates:

- fv vs. pv:  console
  -- fv: ttyS0
  -- pv: hvc0
- kvm vs. xen: default network interface name(which I forgot in my pr, just corrected it and pushed a new commit).
  -- kvm: enp1s0
  -- xen pv&fv: eth0

Reply:
Good then.